Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAO.java

```diff
@@ -161,4 +167,189 @@ public List queryAuditRepositoryOnCriteria(String eUserId, String ePatientId, Da
         }
         return queryList;
     }
+
+    /**
+     * This method does a query to database to get the Audit Log Messages based different options
+     *
+     * @param messageId
+     * @param relatesTo
+     * @return List
+     */
+    public List queryAuditRecords(String messageId, String relatesTo) {
+
+        Session session = null;
+        List<AuditRepositoryRecord> queryList = null;
+        try {
+            SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+            session = sessionFactory.openSession();
+
+            LOG.info("Getting Record for Audit Viewer ");
+
+            // Build the criteria
+            Criteria queryCriteria = session.createCriteria(AuditRepositoryRecord.class);
+
+            if (NullChecker.isNotNullish(messageId)) {
+                if (messageId.startsWith(NhincConstants.WS_SOAP_HEADER_MESSAGE_ID_PREFIX)) {
+                    queryCriteria.add(Restrictions.eq("messageId", messageId));
+                } else {
+                    queryCriteria.add(Restrictions.eq("messageId", NhincConstants.WS_SOAP_HEADER_MESSAGE_ID_PREFIX + messageId));
+
```

Comment:
Alot of unnecessary spacing around here.  Can remove the empty lines such as line 196, 198, and 203